config/config.go

```diff
@@ -186,19 +196,50 @@ func FromViper(v *viper.Viper) (*Config, error) {
 		cfg.Walk = walk.Stdin.String()
 	}
 
+	// set git-based tree root command if the walker is Git
+	if cfg.Walk == walk.Git.String() {
+		cfg.TreeRootCmd = "git rev-parse --show-toplevel"
+	}
```

Comment:
This looks like it is overriding the user-provided `tree-root-cmd` setting when in a git tree?

Reply:
Good catch.